frontend/components/login/index.js

```diff
@@ -186,6 +187,7 @@ class Login extends Component {
                 <div className="row form-row form-row__footer">
                     <div className="col-xs-12">
                         <button
+                            id="submit"
```

Comment:
submit => "submit-button", name in the same logical style

Reply:
Fixed